tools/get_ir_is_by_model_id.py

```diff
@@ -0,0 +1,113 @@
+"""
```

Comment:
Could we give a more readable name for this tool than `get_ir_is_by_model_id.py?`